interface/frontend/index.html

```diff
@@ -100,8 +100,8 @@ <h3 class="card-title">{{ selected.patient_id }}</h3>
     methods: {
       fetchData: function() {
         var vm = this;
-        this.$http.get('/api/images/').then(
-          function(response) {
+        this.$http.get('/api/images/available').then(
```

Comment:
Hm, I think this should use ``{% url … %}`` ?

Reply:
Whole <script> is under {% verbatim %}. Even if verbatim is omitted from script section, is it possible to use {% url %} inside Vue ?